src/main/Extensions/WindowsControlPanel/Contract/WindowsControlPanelItemsRepository.ts

```diff
@@ -0,0 +1,5 @@
+import { WindowsControlPanelItem } from "../WindowsControlPanelItem";
+
+export interface WindowsControlPanelItemsRepository {
+    retrieveControlPanelItems(alreadyKnownItems: WindowsControlPanelItem[]): Promise<WindowsControlPanelItem[]>;
+}
```

Comment:
The `Contract` folder is used to expose contracts to the outside of the module or extension. As this interface is only used inside of this extension I suggest renaming the Contract folder to something else, or even easier just move it into the root of the extension folder.

Reply:
Done. Since the interface has the same name as the class, the file now is not named like the interface.